resources/views/auth/emails/invite.blade.php

```diff
@@ -2,65 +2,71 @@
 <?php
 
 $style = [
-    /* Body ------------------------------ */
+  /* Body ------------------------------ */
 
-    'body_action' => 'width: 100%; margin: 30px auto; padding: 0; text-align: center;',
-    'body_sub' => 'margin-top: 25px; padding-top: 25px; border-top: 1px solid #EDEFF2;',
+  'body_action' => 'width: 100%; margin: 30px auto; padding: 0; text-align: center;',
+  'body_sub' => 'margin-top: 25px; padding-top: 25px; border-top: 1px solid #EDEFF2;',
 
-    /* Type ------------------------------ */
-    'anchor' => 'color: #3869D4;',
-    'paragraph' => 'margin-top: 0; color: #74787E; font-size: 16px; line-height: 1.5em;',
-    'paragraph-sub' => 'margin-top: 0; color: #74787E; font-size: 12px; line-height: 1.5em;',
+  /* Type ------------------------------ */
+  'anchor' => 'color: #3869D4;',
+  'paragraph' => 'margin-top: 0; color: #74787E; font-size: 16px; line-height: 1.5em;',
+  'paragraph-sub' => 'margin-top: 0; color: #74787E; font-size: 12px; line-height: 1.5em;',
 
-    /* Buttons ------------------------------ */
-    'button' => 'display: block; display: inline-block; width: 200px; min-height: 20px; padding: 10px;
+  /* Buttons ------------------------------ */
+  'button' => 'display: block; display: inline-block; width: 200px; min-height: 20px; padding: 10px;
                  background-color: #3869D4; border-radius: 3px; color: #ffffff; font-size: 15px; line-height: 25px;
                  text-align: center; text-decoration: none; -webkit-text-size-adjust: none;',
 
-    'button-blue' => 'background-color: #3869D4;',
+  'button-blue' => 'background-color: #3869D4;',
 
-    'font_family' => 'font-family: Arial, \'Helvetica Neue\', Helvetica, sans-serif;',
+  'font_family' => 'font-family: Arial, \'Helvetica Neue\', Helvetica, sans-serif;',
 ];
 ?>
 @section('content')
-<p>Melding dat u toegevoegd werd aan de dienst "{{ $service }}" op het openingsurenplatform </p>
+@if (count($services) > 1)
+<p>Melding dat u toegevoegd werd aan volgende diensten op het openingsurenplatform:</p>
+<ul>
+  @foreach ($services as $service )
+  <li>{{$service->label}}</li>
```

Comment:
Nitpick: Most of the code style in this file uses `{{ $string }}` and here `{{$string}}` is used (spacing).